src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,52 @@
+.nav {
```

Comment:
file name should be the same as bem block name

Reply:
it's almost fixed. Change a file name from cards.scss to card.scss